src/coreclr/src/nativeaot/Directory.Build.props

```diff
@@ -0,0 +1,104 @@
+<Project>
+  <Import Project="../Directory.Build.props" />
+
+  <PropertyGroup>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <GenerateResxSourceOmitGetResourceString>true</GenerateResxSourceOmitGetResourceString>
+    <GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>
+    <EnsureRuntimePackageDependencies>false</EnsureRuntimePackageDependencies>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+
+    <DebugType>Portable</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+
+    <OutputPath>$(BinDir)/aotsdk/</OutputPath>
+    <Configurations>Debug;Release;Checked</Configurations>
+    <Platforms>x64;x86;arm;arm64</Platforms>
+
+    <Platform Condition=" '$(Platform)' == '' ">$(TargetArchitecture)</Platform>
+    <Platform Condition=" '$(Platform)' == 'armel' ">arm</Platform>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- This prevents the default MsBuild targets from referencing System.Core.dll -->
+    <AddAdditionalExplicitAssemblyReferences>false</AddAdditionalExplicitAssemblyReferences>
+    <!-- These prevent the default MsBuild targets from referencing System.dll and mscorlib.dll -->
+    <ExcludeMscorlibFacade>true</ExcludeMscorlibFacade>
+    <NoStdLib>true</NoStdLib>
+    <NoCompilerStandardLib>true</NoCompilerStandardLib>
+    <SubsystemVersion>6.00</SubsystemVersion>
+    <RuntimeMetadataVersion>v4.0.30319</RuntimeMetadataVersion>
+    <Utf8Output>true</Utf8Output>
+    <HighEntropyVA>true</HighEntropyVA>
```

Comment:
I suspect  many of these properties are not needed anymore. We can clean it up later.